src/routes/+page.svelte

```diff
@@ -1,4 +1,4 @@
 <h1>Nv7's Projects</h1>
 <p class="lead">
-  Welcome to Nv7's Projects! This is where Nv7 keeps all of his fun or interesting programs! This page is being rendered via <a href="https://kit.svelte.dev/">SvelteKit</a> and written in <a href="https://www.typescriptlang.org/">TypeScript</a>. Thanks for check out Nv7's Projects and be sure to try some of the projects!
+  Welcome to Nv7's Projects! This is where Nv7 keeps all of his fun and interesting programs! This page is being rendered via <a href="https://kit.svelte.dev/">SvelteKit</a> and is written in <a href="https://www.typescriptlang.org/">TypeScript</a>. Thanks for checking out Nv7's Projects and make sure to try some of the projects!
```

Comment:
I meant it to be `fun or interesting` because some of them are fun and some of them are interesting

Reply:
Alright